gis/orfeotoolbox/Portfile

```diff
@@ -26,12 +26,12 @@ homepage            https://www.orfeo-toolbox.org/
 master_sites        https://www.orfeo-toolbox.org/packages/
 
 distname            OTB-${version}
-use_xz              yes
+use_xz              no
```

Comment:
this line should just be removed as it defaults to `no`

Reply:
I deliberately kept it in case they add xz file by next release, which would save some 100 MB. Initially they missed the tarball at all with the release, only published some binary packages, so I'm glad I could achieve that much.